docs/rules/no-useless-computed-key.md

```diff
@@ -0,0 +1,46 @@
+# Disallow unnecessary computed property keys on objects (no-useless-computed-key)
+
+It's unnecessary to use computed properties with literals such as:
+
+```js
+var foo = {["a"]: "b"};
+```
+
+The code can be rewritten as:
+
+```js
+var foo = {"a": "b"};
+```
+
+## Rule Details
+
+This rule aims to flag unnecessary usages of computed property keys.
```

Comment:
To remain consistent with current doc updates, what do you think of this?

`This rule disallows unnecessary usage of computed property keys.`